Benchmarks/Benchmarks/HummingbirdBenchmarks/Benchmarks.swift

```diff
@@ -12,10 +12,12 @@
 //
 //===----------------------------------------------------------------------===//
 
-import NIOCore
+@testable import Hummingbird
+import Benchmark
 
-public func randomBuffer(size: Int) -> ByteBuffer {
-    var data = [UInt8](repeating: 0, count: size)
-    data = data.map { _ in UInt8.random(in: 0...255) }
-    return ByteBufferAllocator().buffer(bytes: data)
+let benchmarks = {
+    trieRouterBenchmarks()
+    routerBenchmarks()
+    // The HTTP benchmarks are so variable I'm not sure they are worthwhile running
+    //http1Benchmarks()
```

Comment:
Any idea why they're so diverse?

Reply:
I don't know if it is a timing issue causing more allocations. I would prefer if we could get these more stable as they test the whole pipeline more thoroughly